### PR TITLE
Chore: Expose icons and add icon documentation

### LINKED ISSUE #81142: Add new Icons into Saga
**What is the chore?**:
We need to add 3 new icons to the system:
- Unarchive unarchive custom made (unarchive.svg) - [Figma link](https://www.figma.com/file/htVIFrTq8e7sx8Olo7L6kZ/Saga-Components--Dark-theme?type=design&node-id=10845-14025&mode=design&t=Ij1jRqphOsKztnEN-4).
- Globe ([globe.svg](https://github.com/grafana/grafana/blob/main/public/img/icons/unicons/globe.svg)) - [Figma link](https://www.figma.com/file/htVIFrTq8e7sx8Olo7L6kZ/Saga-Components--Dark-theme?type=design&node-id=20527-963&mode=design&t=Ij1jRqphOsKztnEN-4).
- Archive ([archive-alt.svg](https://github.com/grafana/grafana/blob/main/public/img/icons/unicons/archive-alt.svg)) - [Figma link](https://www.figma.com/file/htVIFrTq8e7sx8Olo7L6kZ/Saga-Components--Dark-theme?type=design&node-id=20737-94&mode=design&t=Ij1jRqphOsKztnEN-4).
- we also need to improve our documentation for how to add icons in grafana
  - investigate whether any documentation exists already (we think not)
  - add documentation to https://github.com/grafana/grafana/tree/main/public/img/icons
  - link to it from saga

**Is there anything else we need to know?**:
N/A

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- exposes 3 new icons (`archive-alt`, `globe` and `unarchive`)
- adjusts the `application-observability` icon to not have a clashing class definition
- adds a rough draft of documentation for adding a new icon

**Why do we need this feature?**

- so it's clear how to expose new icons

**Who is this feature for?**

- grafana devs

**Which issue(s) does this PR fix?**:

Fixes #81142 

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/img/icons/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# How to add a new icon
+
+- Add the new icon svg to the `unicons/` directory
```
Comment (ashharrison90) [inline public/img/icons/README.md:3]: this is almost certainly wrong, but what should we put here?

essentially we have to explain the logic behind: https://github.com/grafana/grafana/blob/ash/81142/packages/grafana-ui/src/components/Icon/utils.ts#L13

`Icon` defaults to the `unicons` directly hence why most icons get put in here, but we probably need some logical definition of what icons fit where.

wdyt @grafana/grafana-frontend-platform ?

Comment (Clarity-89) [reply public/img/icons/README.md:3]: Wasn't it so if the icon is the one from [here](https://iconscout.com/unicons/solid-icons), it'd be added to the `unicons` folder. If it's a custom made icon, then it'd be in the `custom` folder? Also we'd probably get rid of the `solid` and `mono` folders and just use the unicons one.

Comment (Aires-Grafana) [reply public/img/icons/README.md:3]: I agree with @Clarity-89.

File: public/img/icons/unicons/unarchive.svg
Comment (Aires-Grafana) [inline public/img/icons/unicons/unarchive.svg:1]: @ashharrison90 this icon should be left inside the custom folder, not the unicon folder as it was custom-made.

File: public/img/icons/unicons/application-observability.svg
Comment (Aires-Grafana) [inline public/img/icons/unicons/application-observability.svg:1]: @ashharrison90 why is this icon in the unicons folder? this is a custom-made icon from Grafana.

**Review by Aires-Grafana — COMMENTED** [review]
Left some comments about where to put the icons regarding to their correct folder location.

File: public/img/icons/unicons/unarchive.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (Clarity-89) [reply public/img/icons/unicons/unarchive.svg:1]: I think `fill` is unnecessary as we set it via the CSS applied to SVGs.

**Review by Clarity-89 — APPROVED** [review]
🚀

**Review by JoaoSilvaGrafana — APPROVED** [review]
👏

**Pull request merged**